src/main/java/org/carlmontrobotics/robotcode2023/subsystems/Arm.java

```diff
@@ -169,11 +119,13 @@ public double getKg() {
     }
 
     public double getArmPos() {
-        return MathUtil.clamp(MathUtil.inputModulus(armEncoder.getPosition(), ARM_ANGLE_MIN, ARM_ANGLE_MAX), ARM_ANGLE_MIN + ARM_ANGLE_TOLERANCE, ARM_ANGLE_MAX - ARM_ANGLE_TOLERANCE);
+        return MathUtil.inputModulus(armEncoder.getPosition(), ARM_DISCONTINUITY_RAD,
+                ARM_DISCONTINUITY_RAD + 2 * Math.PI);
     }
 
     public double getWristPos() {
-        return MathUtil.clamp(MathUtil.inputModulus(wristEncoder.getPosition(), WRIST_ANGLE_MIN, WRIST_ANGLE_MAX), WRIST_ANGLE_MIN + WRIST_ANGLE_TOLERANCE, WRIST_ANGLE_MAX - WRIST_ANGLE_TOLERANCE);
+        return MathUtil.inputModulus(wristEncoder.getPosition(), ARM_DISCONTINUITY_RAD,
+                ARM_DISCONTINUITY_RAD + 2 * Math.PI);
     }
```

Comment:
Spot the bug

Reply:
Nope, mod is correct

Reply:
[f6ce832](https://github.com/DeepBlueRobotics/RobotCode2023/pull/6/commits/f6ce8328508e10a846847eb6a8246134207070b9) fix